OpenRA.Mods.Common/Traits/World/ActorSpawnManager.cs

```diff
@@ -0,0 +1,118 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Controls the spawning of actors. Attach this to the world actor.")]
```

Comment:
Can you please change this to something like "Controls spawning of map creep actors"?  There are other types of actor spawning in the game, so this could otherwise be confusing.

Reply:
Yeah, but also those actors aren't creep only, depends on what is set in Owner of the manager, so "Controls spawning of the map actors"?

Reply:
I would go with "Controls the spawning of specified actor types" to keep the distinction vs e.g. `SpawnMpUnits` clear.